app/locales/nl.json

```diff
@@ -1,146 +1,150 @@
-  "Number reported in COVID-19 infected patients": "Aantal gemeld bij met COVID-19 geïnfecteerde patiënten",
-  "Number reported": "Aantal gemeld",
-  "Reports with symptoms": "Rapporteringen met symptomen",
-  "Reports": "Rapporteringen",
-  "In close contact with symptoms": "In nauw contact met symptomen",
-  "In close contact without symptoms": "In nauw contact zonder symptomen",
-  "Close contact with someone who has tested positive": "Nauw contact met iemand die positief getest is",
-  "Tested negative": "Negatief getest",
-  "Map": "Kaart",
-  "API": "API",
-  "Statistics": "Statistieken",
-  "Download the dataset as CSV": "Download de dataset in CSV-formaat",
-  "COVID-19 infected": "COVID-19 geïnfecteerden",
-  "people have reported that they have tested positive for COVID-19 and experience symptoms.": "mensen gemeld dat ze positief zijn getest op COVID-19 en symptomen hebben.",
-  "Symptoms": "Symptomen",
-  "Test results": "Testresultaten",
-  "people have reported that they have been tested for COVID-19.": "mensen gemeld dat ze zijn getest op COVID-19.",
-  "Close contacts": "Nauwe contacten",
-  "people have reported that they have been in close contact with a person who was tested positive for COVID-19.": "mensen gemeld dat ze nauw contact hebben gehad met een persoon die positief is getest op COVID-19.",
-  "Key figures": "Kerncijfers",
-  "Of those who reported that they have been in close contact with someone that tested positive for COVID-19, have:": "Van degenen die hebben gemeld dat ze in nauw contact staan met een persoon met een COVID-19-infectie, hebben:",
-  "reported that they experience the two most common symptoms those who are infected experience.": "gemeld dat ze de twee meest voorkomende symptomen hebben die geïnfecteerde mensen hebben.",
-  "reported that they experience the three most common symptoms those who are infected experience.": "gemeld dat ze de drie meest voorkomende symptomen hebben die geïnfecteerde mensen hebben.",
-  "All submissions": "Alle ingezonden rapporten",
-  "Everyone who has submitted a report, also has:": "Van alle mensen die hebben gerapporteerd:",
-  "Total number of reports": "Totaal aantal meldingen",
-  "Positive test results and close contact": "Positieve testresultaten en nauw contact",
-  "Clarifications data basis": "Verduidelijkingen gegevensbasis",
-  "Questions about close contact were added on 16 March at. 10:30 p.m.. Some symptoms were not part of the form from the beginning:": "Vragen over nauw contact zijn op 16 maart toegevoegd op. 10:30 p.m .. Sommige symptomen maakten vanaf het begin geen deel uit van het formulier:",
-  "Symptom": "Symptoom",
-  "Added to form": "Toegevoegd aan formulier",
-  "https://who.int": "https://who.int",
-  "To WHO.int": "To WHO.int",
-  "Are you from": "Kom je uit",
-  "Norway": "Noorwegen",
-  "Netherlands": "Nederland",
-  "Slovakia": "Slowakije",
-  "United Kingdom": "Verenigd Koninkrijk"
+	"Postal code": "Postcode",
```

Comment:
Could we change this back to using spaces?

Reply:
Yes, sorry.